split.Console/Interfaces/IPlayerInterface.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a very rough draft. It will eventually be implemented by an 
+    /// object that will encapsulate all player-related behavior. 
+    /// </summary>
+    public interface IPlayerInterface : ICardInterface
```

Comment:
Why would it extend `ICardInterface`? Wouldn't I have hand instead have a function that returns a `List<ICardInterface>`? That isn't an _implementation_ of `ICardInterface` simply a use.